pkgs/top-level/all-packages.nix

```diff
@@ -3413,6 +3413,8 @@ with pkgs;
 
   azure-cli = callPackage ../tools/admin/azure-cli { };
 
+  azure-cli-extensions = recurseIntoAttrs azure-cli.extensions;
```

Comment:
Are we calling callPackages for the extensions?

Reply:
Good point, we definitely should. You'd say this is the right place for it?

Reply:
Yeah, seems better in all-packages.nix. This way, we can still use the passthru to build a CLI with all extensions like:
```
 nix build --impure --expr 'with (import ./. {}); azure-cli.withExtensions (lib.attrValues azure-cli.extensions)'
 ```
 which will fail if we use `callPackage` inside `azure-cli/default.nix`, as `callPackage` adds something that hasn't go a pname.

Reply:
Never mind, can't be added in all-packages.nix.